templates/jvb-statefulset.yaml

```diff
@@ -80,6 +80,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: status.hostIP
+        - name: JVB_ADVERTISE_IPS
+          valueFrom:
+            fieldRef:
+              fieldPath: status.hostIP
```

Comment:
I don't think this is a sane default (or that there is any sane default). e.g. on EKS, even on public subnets we don't have the IP that would be referred to here

Reply:
So this is only the case if we set `provideNodeAddressAsPublicIP`. It's specific cases where we are not behind NAT, which I dont think is the case on EKS.

Reply:
Right, I'd missed the gating sorry.

In which case my comment about `JVB_ADVERTISE_PRIVATE_CANDIDATES` might be wrong too. Potentially that wants to be set to the value of `provideNodeAddressAsPublicIP` or at the very least configurable

Reply:
I was planning to override it via extra env vars, but you are right it will be cleaner to directly provide a variable to do so.